RecoTracker/MeasurementDet/plugins/TkGluedMeasurementDet.cc

```diff
@@ -139,7 +152,8 @@ bool TkGluedMeasurementDet::measurements( const TrajectoryStateOnSurface& stateO
 				      (theMonoDet->hasAllGoodChannels() || 
 				       testStrips(stateOnThisDet,gluedPlane,*theMonoDet)
 				       )
-				      ) /*Mono OK*/ || 
+				      ) /*Mono OK*/ 
+                                     && // was || 
```

Comment:
This change is responsible for small changes in RECO (because the change does not depend on the on/off of the mitigation).